yoonsang/Week 2/acmicpc1181.c

```diff
@@ -0,0 +1,53 @@
+#include <stdio.h>
+#include <string.h>
+char a[20000][50], temp[20000][50];
+int alen[20000],templen[20000],M;
+int vocajud(char a[], char b[]){
+    if(strlen(a)<strlen(b)) return 1;
+    else if(strlen(a)>strlen(b)) return 0;
+    for(int i=0;i<strlen(a);i++){
+        if(a[i]<b[i]) return 1;
+        else if(a[i]>b[i]) return 0;
+    }
+    return 0;
+}
+int msort(int L, int R){
+    if(L>=R) return 0;
+    M=(L+R)/2;
+    msort(L, M);
+    msort(M+1, R);
+    for(int i=L, l=L, r=M+1; l!=M+1 || r!=R+1; i++){
+        if ((r!=R+1 && l<=M && vocajud(a[l], a[r])) || r==R+1) {
+          for(int j=0;j<alen[l];j++)
+            temp[i][j] = a[l][j];
+          templen[i]=alen[l];
+          printf("temp: %s / a: %s, l: %d\n", temp[i], a[l], l);
+          l++;
+        } else {
+          for(int j=0;j<alen[r];j++)
+            temp[i][j] = a[r][j];
+          templen[i]=alen[r];
+          printf("temp: %s / a: %s, r: %d\n", temp[i], a[r], r);
+          r++;
+        }
+        printf("L: %d / i: %d / l: %d / r: %d %s %s %s\n", L,i,l,r,temp[i], a[l], a[r]);
+    }
+    for (int i=L; i<=R; i++) {
+      for(int j=0;j<templen[i];j++)
+        a[i][j] = temp[i][j];
```

Comment:
이 부분도, character를 하나하나 옮기시는 것 보다는 
strcpy를 이용해서
```suggestion
         strcpy(a[i], temp[i])
         alen[i] = templen[i] 
```
이런 식으로 해주시면 훨씬 편하실 것 같아요!